scripts/exregional_run_gridstatvx.sh

```diff
@@ -116,6 +117,7 @@ export MET_CONFIG
 export OBS_DIR
 export VAR
 export MODEL
+export NET
```

Comment:
I'm fine with NET - does it stand for something? (I should probably know this)

Reply:
I actually don't know what it stands for, but I noticed that the post run script uses it to name the post output. It is set and described in config_defaults.sh. I figured it would be good to use the preexisting post settings instead of creating a new variable...but that can definitely be changed!

Reply:
Ha! Ok. Totally fine with reusing the variable. Was just curious what it was short for!